eng/pipelines/templates/steps/build-and-test.yml

```diff
@@ -54,7 +57,7 @@ steps:
     env:
       AZURE_VERSION_OVERRIDE_TESTS: ${{ parameters.TestVersionSupport }}
     condition: and(succeeded(), ne(variables['TestFromSource'], 'true'))
-    continueOnError: ${{ eq(variables['LatestJdkVersion'], variables['JavaTestVersion']) }}
+    continueOnError: ${{ parameters.ContinueLatestBuildOnError }}
```

Comment:
```suggestion
    continueOnError: eq(variables['LatestJdkVersion'], variables['JavaTestVersion'])
```
Does that syntax work correctly?

Reply:
This results in a syntax error

![image](https://user-images.githubusercontent.com/48699787/199354933-be26989c-fc6b-4f36-8e6b-8c55ffbd724f.png)